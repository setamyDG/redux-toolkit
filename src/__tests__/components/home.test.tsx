import Home from '@views/home';
import React from 'react';
import { render } from '../__utils__/test-utils';

describe('Home', () => {
  it('should render Home component', () => {
    const { container } = render(<Home />);
    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
