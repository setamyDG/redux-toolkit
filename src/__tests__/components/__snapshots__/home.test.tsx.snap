// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render Home component 1`] = `
<div>
  <section
    class="sectionstyled-sc-1ae8sz4-0 cXxQmw"
  >
    <div
      class="containerstyled-sc-1yi0ay3-0 ksPUcq"
    >
      <h1>
        home
      </h1>
    </div>
  </section>
</div>
`;
